File: bakalarkaBE/bakalarkaBE/Models/Liekyzdielanie.cs
﻿using System;
using System.Collections.Generic;

namespace bakalarkaBE.Models;

public partial class Liekyzdielanie
{
    public string Zdielajuci { get; set; } = null!;

    public string Cielovy { get; set; } = null!;

    public string Registracnecislo { get; set; } = null!;

    public string Rodnecislo { get; set; } = null!;

    public DateOnly Datumod { get; set; }

    public DateOnly DatumdoZdielanie { get; set; }

    public virtual Doktor CielovyNavigation { get; set; } = null!;

    public virtual Pacientovelieky Pacientovelieky { get; set; } = null!;

    public virtual Doktor ZdielajuciNavigation { get; set; } = null!;
}

File: FE/src/app/models/LiekyZdielanie.ts
import {Doktor} from "./Doktor";
import {PacientLieky} from "./PacientLieky";

export class LiekyZdielanie {
  zdielajuci:string = "";
  cielovy:string = "";
  registracnecislo:string = "";
  rodnecislo:string = "";
  datumod:string = "";
  datumdoZdielanie:string = "";
  cielovyNavigation?:Doktor;
  pacientovelieky:PacientLieky = new PacientLieky();
  zdielajuciNavigation?:Doktor;
}
